src/main/java/com/sinthoras/visualprospecting/Utils.java

```diff
@@ -57,6 +57,15 @@ public static boolean isJourneyMapInstalled() {
     public static boolean isXaerosWorldMapInstalled() {
         return Loader.isModLoaded("XaeroWorldMap");
     }
+    
+    public static boolean isVoxelMapInstalled() {
+    	try {
+    		Class.forName("com.thevoxelbox.voxelmap.litemod.LiteModVoxelMap");
```

Comment:
This is required because of LiteLoader?

Reply:
Yes, calling `Loader.isModLoaded()` is not possible for LiteLoader and its mods